dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/statelisteners/ElasticListener.java

```diff
@@ -208,9 +208,10 @@ public void afterBulk(long executionId, BulkRequest request,
                     LOGGER.error("Bulk [{}] executed with failures", executionId);
                     for (BulkItemResponse bulkItemResponse : response.getItems()) {
                         if (bulkItemResponse.isFailed()) {
-                            Throwable failure = bulkItemResponse.getFailure().getCause().getCause();
-                            LOGGER.error("Item {} in bulk [{}] executed with failure", bulkItemResponse.getItemId(), executionId, failure);
-                            afterBulkFailures.add(failure);
+                            final Failure failure = bulkItemResponse.getFailure();
+                            final Throwable throwable = failure.getCause().getCause();
```

Comment:
The code robot noticed that `throwable` was unused.  Was it intended to be used somewhere?

Reply:
Aargh, good catch.